gradle.properties

```diff
@@ -4,20 +4,20 @@ kotlin.code.style=official
 org.gradle.jvmargs=-Xmx3G
 # Fabric
 minecraft_version=1.16.4
-yarn_mappings=1.16.4+build.1:v2
-loader_version=0.9.3+build.207
-loom_version=0.5-SNAPSHOT
+yarn_mappings=1.16.4+build.9
+loader_version=0.11.1
```

Comment:
086 asked me not to bump to loader so less people are forced to update, in previous prs

Reply:
Unfortunately, we need to bump the loader from 0.9.3 to at least 0.10.1 to get the fabricmc mixin 0.8.2. I would consider dropping down to near 0.10.x We shouldn't use Mixin 0.8.1. Also, 0.10.1 allows for flexibility around dependency overwriting which may be needed by some mods if jar files conflict. It's also the recommended version from Fabric going forward, additionally includes https://github.com/FabricMC/fabric-loader/commit/5cd3b1b1bd3f12525bc5245a6edaeb657c146ebe which from what I understand will give a better message when Fabric detects a mod using newer language (like we are now Java 11).

Reply:
As long as it's not unnecessary, that's fine. Looks good to me

Reply:
And yes, Mixin 0.8.1 is a bit weird and unstable

Reply:
I'm thinking 0.10.x is safe, but I'll probably need to survey what mods out there that people use in combination.

Lithium uses 0.11.1 in dev and 0.10.6 tagged for stable. I may not know the consequences fully, so there may be a good reason.

Reply:
I'm confident this'll work fine - KAMI isn't using anything advanced from loader anyways.